apps/meteor/app/utils/lib/mimeTypes.ts

```diff
@@ -12,7 +12,11 @@ const getExtension = (param: string): string => {
 	return !extension || typeof extension === 'boolean' ? '' : extension;
 };
 
-const getMimeType = (fileName: string): string => {
+const getMimeType = (mimetype: string, fileName: string): string => {
+	if (mimetype && mimetype !== 'application/octet-stream') {
```

Comment:
I believe this could cause other issue :thinking: 

As an example, check the `ico` MIME type definition [above](https://github.com/RocketChat/Rocket.Chat/pull/32763/files#diff-e64159623d12c88eb4700ba6e62fc7a89517ee0603d165af21938f9fd899e582R7). We want it to be mapped to the `image/x-icon` MIME type. With the change you suggested here, we'll prefer the MIME type we get from busboy (which is not `application/octet-stream`, but actually `image/vnd.microsoft.icon`) -- that could hurt users that have blocked the `image/x-icon` MIME type :thinking:

Please check #32764 , I tried another approach that also fixes this issue

Reply:
@matheusbsilva137 can you review the code now?